main.js

```diff
@@ -0,0 +1,51 @@
+
+    const form = document.querySelector("form");
+    form.addEventListener("submit", (event) => {
+     event.preventDefault()
+     triviaQuesitons()
+ })
+
+
+ const triviaQuesitons = () => {
+
+fetch('https://opentdb.com/api.php?amount=10')
+.then((response) => response.json())
+.then((data) => console.log(data))
```

Comment:
Your code for showing the cards isn't in your callback here. So that's why data is undefined, because you're basically just console.logging data and then finishing up with it.